src/example/FluxOnly/FluxOnly.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {Store, Url, ActionCreator} from '../..';
+import styles from './FluxOnly.css';
+
+const Switch = React.createClass({
+
+  getInitialState() {
+    return {
+      query: Store.getQuery()
+    };
+  },
+
+  componentDidMount() {
+    this.unsubscribe =
+      Store.addThrottledChangeListener(this.onChange, 50);
+  },
+
+  componentWillUnmount() {
+    this.unsubscribe();
+  },
+
+  switchComponents() {
+    let query = 'Nothing';
+
+    if ('switch' in this.state.query) {
+      query = this.state.query.switch;
+    }
+    return query;
+  },
+
+  onChange() {
+    this.setState({
+      query: Store.getQuery()
+    });
+  },
+
+  render() {
+    return (
+      <div className={styles.content}>
+        <p>Current state is: {this.switchComponents()} Clicked</p>
+      </div>
+    );
+  }
+});
+
+const FluxWithUrls = React.createClass({
+
+  render() {
+    return (
+      <div className={styles.fluxonly}>
+        <div>
+          <h2>Using component-router's Urls</h2>
+        </div>
+        <div>
+          <Url query={{switch: 'First'}}>
+            Render First Component
+          </Url>
+          <Url query={{switch: 'Second'}}>
+            Render Second Component
+          </Url>
+          <Switch />
+        </div>
+      </div>
+    );
+  }
+
+});
+
+const PureFlux = React.createClass({
+
+  getInitialState() {
+    return {
+      query: Store.getQuery()
```

Comment:
For "smart" components, I usually make whole store data as state to avoid extra nesting. So just ` ruturn Store.getQuery()`